src/helpers/mapbox-layers.js

```diff
@@ -464,6 +464,22 @@ const data = [
       "circle-stroke-width": 2,
     },
   },
+  {
+    id: "evtlocations",
```

Comment:
Does the API return these polygons as points? Or does MapboxGL convert them to points? Maybe add a code comment so that this info won't be lost in this PR.

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/187/commits/1ec07ac48e8f40ef8e0a685016b8e31f15716309